cli/src/commands/describe.rs

```diff
@@ -86,7 +86,7 @@ pub(crate) struct DescribeArgs {
     /// allow the message to be edited afterwards.
     #[arg(long)]
     edit: bool,
-    /// Reset the author to the configured user
+    /// Reset the author to the configured user and update author timestamp
     ///
     /// This resets the author name, email, and timestamp.
```

Comment:
Maybe we can combine these if we should mention timestamp in the short description? Something like "Reset the author name, email, and timestamp" (without "to the configured user") sounds good to me. If needed, "to the configured user" can be moved to the detailed doc.

Reply:
That sounds reasonable to me